migrations/2024-12-17-144959_community-post-tags/up.sql

```diff
@@ -0,0 +1,26 @@
+-- a tag for a post, valid in a community. created by mods of a community
+CREATE TABLE tag (
+    id serial PRIMARY KEY,
+    ap_id text NOT NULL UNIQUE,
+    name text NOT NULL,
+    published timestamptz NOT NULL DEFAULT now(),
+    updated timestamptz,
+    deleted timestamptz
```

Comment:
The standard for deletes is a deleted bool column, which then sets the `updated` column to the delete date. So no need for two timestamptz columns.